net/third_party/quiche/src/quiche/quic/core/crypto/crypto_protocol.h

```diff
@@ -194,8 +194,26 @@ const QuicTag kMAD0 = TAG('M', 'A', 'D', '0');   // Ignore ack delay
 const QuicTag kMAD2 = TAG('M', 'A', 'D', '2');   // No min TLP
 const QuicTag kMAD3 = TAG('M', 'A', 'D', '3');   // No min RTO
 const QuicTag k1ACK = TAG('1', 'A', 'C', 'K');   // 1 fast ack for reordering
+#if defined(USE_COBALT_CUSTOMIZATIONS)
+const QuicTag kAKD1 = TAG('A', 'K', 'D', '1');   // 10 packets received before
+                                                 // acking
+const QuicTag kAKD2 = TAG('A', 'K', 'D', '2');   // 20 packets received before
+                                                 // acking
+#endif  // defined(USE_COBALT_CUSTOMIZATIONS)
 const QuicTag kAKD3 = TAG('A', 'K', 'D', '3');   // Ack decimation style acking
                                                  // with 1/8 RTT acks.
+#if defined(USE_COBALT_CUSTOMIZATIONS)
+const QuicTag kAKD5 = TAG('A', 'K', 'D', '5');   // 40 packets received before
```

Comment:
Why do we skip AKD4?

Reply:
It would be nice if we could follow a pattern for determining what number the tag corresponds to without looking at the comment as is done sometimes in these naming schemes—i.e. having `AKD<N>` with `2^N * 10` packets received before acking. That pattern would of course overlap with AKD3 if we don't change the prefix letters and not give us all of the current values you've added (instead we'd have 80/160/320/... on the high end), so I'd understand if that's not something you'd want to do, but it would make it easier for people to get accustomed to.

I don't expect it to be widely used, so we don't have to think about the absolutely best naming scheme here, and it's not a big deal if we stick with what you have here